### PR TITLE
Payment pages form

### PR DESCRIPTION
**Pull request by maxwofford**

### CODE REVIEW

**Comment by maxwofford** [comment]
@zachlatta This is ready for merge

File: unrejector/views/outcome.erb [hunk 1]
```diff
@@ -1,22 +1,24 @@
...
   <h1 class="white open-sans">The Venue</h1>
   <div class="row">
-    <div class="small-10 small-centered large-4 large-centered columns">
-      <div class="panel">
-        <p></p>
-      </div>
+    <div class="small-10 small-centered large-6 large-centered columns">
+      <a href="http://www.galvanize.com/campuses/san-francisco-soma/">
+        <div class="panel">
+          <h1>Galvanize (SoMa)</h1>
```
Comment (zachlatta) [inline unrejector/views/outcome.erb:12]: Can you change this to "Galvanize SF"?

File: unrejector/views/outcome.erb [hunk 2]
```diff
@@ -1,22 +1,24 @@
 <div class="section banner banner-short text-center">
   <img src="/img/hackedu_logo.png">
   <h3 class="white hackedu-logo"><a href="/">hackEDU's</a></h3>
-  <h1 class="white title">Accept your invite</h1>
+  <h1 class="white title">Welcome to Hack Camp</h1>
 </div>
 <div class="section projects text-center">
   <h1 class="white open-sans">The Venue</h1>
```
Comment (zachlatta) [inline unrejector/views/outcome.erb:7]: Can you change this to "Location"?

File: unrejector/views/outcome.erb [hunk 3]
```diff
@@ -1,22 +1,24 @@
...
-    <div class="small-10 small-centered large-4 large-centered columns">
-      <div class="panel">
-        <p></p>
-      </div>
+    <div class="small-10 small-centered large-6 large-centered columns">
+      <a href="http://www.galvanize.com/campuses/san-francisco-soma/">
+        <div class="panel">
+          <h1>Galvanize (SoMa)</h1>
+        </div>
+      </a>
```
Comment (zachlatta) [inline unrejector/views/outcome.erb:14]: Can you add a map somewhere around here?

File: unrejector/views/payment.erb [hunk 4]
```diff
@@ -6,13 +6,82 @@
...
   <h1 class="white open-sans">The Venue</h1>
   <div class="row">
-    <div class="small-10 small-centered large-4 large-centered columns">
-      <div class="panel">
-        <p></p>
-      </div>
+    <div class="small-10 small-centered large-6 large-centered columns">
+      <a href="http://www.galvanize.com/campuses/san-francisco-soma/">
+        <div class="panel">
+          <h1>Galvanize (SoMa)</h1>
```
Comment (zachlatta) [inline unrejector/views/payment.erb:12]: "Galvanize (SoMa)" -> "Galvanize SF"

File: unrejector/views/payment.erb [hunk 5]
```diff
@@ -6,13 +6,82 @@
...
+        </div>
+      </a>
```
Comment (zachlatta) [inline unrejector/views/payment.erb:14]: Can you add a map somewhere around here?

File: unrejector/views/payment.erb [hunk 6]
```diff
@@ -6,13 +6,82 @@
...
+      <input id="checkbox3" type="checkbox">
+      <label for="checkbox3">I will be dropped from the program if I have 2
+        excused absences.</label>
+      <br>
+      <input id="checkbox4" type="checkbox">
+        <label for="checkbox4">I will bring a laptop.</label>
+      <br>
+      <input id="checkbox5" type="checkbox">
+      <label for="checkbox5">I understand the deadline to pay my tuition for
+        the program is 8pm, Tuesday the 26th.</label>
```
Comment (zachlatta) [inline unrejector/views/payment.erb:63]: Do these checkboxes disable the payment at the bottom unless all of them are checked, as discussed?

**Pull request closed without merging**